addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -137,7 +137,7 @@ ArrayList g_aFastRespawn;
 
 bool g_bBackstabbed[TF_MAXPLAYERS];
 
-int g_iDamage[TF_MAXPLAYERS];
+int g_iDamageInfected[TF_MAXPLAYERS];
```

Comment:
Better to go for `g_iDamageZombie`, `Infected` keyword in code generally only used on special infecteds